Sources/Datasets/Core.swift

```diff
@@ -7,9 +7,19 @@ protocol S5TFDataset where DataLoader: S5TFDataLoader {
     static var train: DataLoader { get }
 }
 
+public enum S5TFSplit {
+    case train
+    case validation
+    case test
+}
+
 public struct S5TFDatasetInfo {
     var name: String
     var version: String
     var description: String
     var homepage: URL
+    var numTrain: Int = 0
+    var numValid: Int = 0
+    var numTest: Int = 0
+    var numFeatures: Int = 0
```

Comment:
Please change this to `numberOf` with no default value. All of these should be constants instead of variables.